_scss/_notes.scss

```diff
@@ -41,6 +41,10 @@ The available classes are:
   > p:only-child, > p:not(:first-child) {
     color: $body-text-color;
   }
+
+  a {
+    text-decoration: underline;
+  }
```

Comment:
This is a belt-and-braces approach. Since these bits of UI are colorful, we shouldn't rely just on color to identify links.